src/sequence.cpp

```diff
@@ -25,6 +25,10 @@ void Sequence::updateLastActivityTime() {
     lastActivityTime = std::chrono::steady_clock::now();
 }
 
+const uint64_t& Sequence::getId() const {
```

Comment:
Uint64 is not a big overhead to copy. Maybe just return `uint64_t`?

Reply:
This also applies to all other functions that take and return const reference to uint64_t which I don't think is neccessary